### PR TITLE
Update interfaces.py

### PR DESCRIPTION
**Pull request by i-zone**
Please remove restriction on 'is_default_page', so that all content could be aliased.

### CODE REVIEW

**Comment by lentinj** [comment]
So the behaviour I see (which I'm guessing is what i-zone is trying to fix) is:-
* Go to your site root, "Add new", "Alias"
* Hit browse on the contenttree widget, you see nothing, or the default page for your site root if you have one (since is_default_page is True there).
...at this point I would like to search through my site and find some content to alias.

This is definitely similar to https://dev.plone.org/ticket/12290 - there plone.app.multilingual is also seemingly looking for content where is_default_page is True or False (whatever the value of that is). The code in question there has been ripped out:-

https://github.com/plone/plone.app.multilingual/commit/a322a12f9b6ad35c854f745307df1780d354b88a

Unfortunately the git history for collective.alias doesn't give many clues as to what the search restriction is trying to do.

TL;DR: +1 for merging this.

**Comment by jaroel** [comment]
Sorry, forgot I had this lying around!
